### DISCUSSION TITLE
Idea: ExpressLRS arming on channel 0 instead of channel 5

### DISCUSSION THREAD

**Discussion by vevere** [ExpressLRS/ExpressLRS#2656] [Ideas]
Hi,
ExpressLRS arming is an important/mandatory feature and it's great.

The choice to use ch5 for this function, however, seems to unnecessarily complicate the use of this protocol a lot while a simpler solution could make this subject much easier for everyone.

Since this function is important and ultimately mandatory, would it be possible (in a future major version) to consider assigning channel 0 to arming?

More and more PWM receivers are available on the market (even if the concept applies identically to flight controllers) and having assigned channel 5 to this function only seems to unnecessarily complicate the understanding and the adoption of this protocol.
For each new receiver/model we must ask ourselves the question of which channel to assign to output 5 of our receivers, this complicates the programming of each new model in our transmitters and can very easily lead to assignment errors between channels or to model programming mistakes.

A large proportion of modelers do not have great computer or development skills (you just have to look at the average age on our airfields) and are not comfortable when we push these concepts a little far in the equipment of our models (ESC, telemetry, mixing, TX programming, etc... ).

So using a channel number not available on the outputs of any receiver or flight controller (that's why I was thinking of channel 0) for the "arming" function could solve simply and elegantly the "problem" for everyone.

it would thus become obvious to everyone that the "ch0" is an essential/mandatory function (by design) but not binding in the use of the protocol.
I understand that a change like this will have a lot of consequences on the entire ecosystem (betaflight, inav, etc...) but the ELRS protocol is still "young" and it is therefore still easily time to make a change such as this one

Thanks for reading.

**Comment by JyeSmith** [comment]
For background, ch5 (aux1) came about because ELRS started with racing quads and using aux1 makes sense.

At the moment there isnt any appetite to change and as you mention a change would essential mean chaos for the existing community.  Any future change will be to remove any reliance on the arm channel.
